data/autoyast_sle15/autoyast_sle_powervm.xml.ep

```diff
@@ -88,13 +88,17 @@
     % foreach (values %$addons) {
         <package><%= lc($_->{name}) %>-release</package>
     % }
+      <package>firewalld</package>
```

Comment:
this is really strange, how existing job was able to enable firewall but wasn't necessary to add the package?
perhaps firewalld is installed by default with AutoYaST when you don't specify it explicitly?

Reply:
I just noticed that it may related with x11 pattern, the tigervnc package is under x11 pattern, x11 was not added when I met the problem. I remove the firewalld package from the ay profile and let's see the result.